### ISSUE TITLE
CDN for pictures

### ISSUE THREAD

**Issue by razzeee** [flathub/website#1453]
So that we can choose the best image resolutions per device size. Maybe even serve webp by default.

**Comment by umairraza96** [comment]
Look into imagekit or cloudinary. I prefer imagekit.

**Comment by razzeee** [comment]
Turns out even nginx can resize images on the fly https://www.nginx.com/blog/responsive-images-without-headaches-nginx-plus/

But doesn't offer a blurhash implementation

Wondering, if @ximion could create blurhash-sums in libappstream?

**Comment by ximion** [comment]
Oh, blurhash looks like a really nice idea to add, especially to any larger images!
I'll need to have a look, maybe we could even simply stop to generate the micro-thumbnails, nobody seems to be using those anyway, and have blurhash sums instead. Given how short those hashes are, it shouldn't even increase the data by that much.

By the way, if `optipng` is available to compose, it will run that on every generated PNG image, usually reducing their size quite substantially ;-)

**Comment by razzeee** [comment]
Added the `optipng` to the initial post, but will have to wait after libappstream port I guess.

**Comment by razzeee** [comment]
wondering, if we can upload our static images to the cdn too https://github.com/flathub-infra/website/tree/main/frontend/public/img

**Comment by razzeee** [comment]
Yeah, then the problem is probably that I reference that one file from css - will take a look at that again at some point
